src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -394,5 +394,6 @@ public static string CertMustHavePrivateKey(string certificateName)
                 errorCode);
 
         public static string CcsRoutingHintMissing = "Either the userObjectIdentifier or tenantIdenifier are missing. Both are needed to create the ccs routing hint. See https://aka.ms/msal-net/ccsRouting. ";
+        public static string ClientCredentialWrongAuthority = "The current authority is targeting the /common or /organizations endpoint which is not reccomended. See https://aka.ms/msal-net-client-credentials for more details.";

```

Comment:
The method is named `IsCommonOrganizationsOrConsumersTenant`. Does it also verify /consumers?
If it not recommended? or not authorized?

Reply:
Yes, Ill add consumers